dotcom-rendering/src/amp/components/ContentABTest.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import sha256 from 'crypto-js/sha256';
+
+const AB_TEST_GROUPS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11] as const;
```

Comment:
That's a good question, I'm not actually sure what the ad manager groups are used for (partitioning requests to pubads by splitting into groups?), and they're definitely a separate usage to this one.

The reason I've gone zero indexed here is because that matches the output of `(x) => x % NUM_GROUPS`. Since the value of these groups is arbitrary (they could equally be `A`, `B`, `C`, ...), keeping them the same as the output of the grouping function should keep it as simple as possible (both here and from the BigQuery analysis side).